core/lib/spree/core/controller_helpers/pricing.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Core
+    module ControllerHelpers
+      module Pricing
+        extend ActiveSupport::Concern
+
+        included do
+          helper_method :current_currency
+        end
+
+        def current_currency
+          current_store.try(:default_currency).presence || Spree::Config[:currency]
```

Comment:
Do we expect current the current_store default currency to return a non-nil blank value (like `""`)? If not, we can simplify this to:

```Ruby
current_store.try!(:default_currency) || Spree::Config[:currency]
```